### PR TITLE
Add validate() to parse.cc fuzzer

### PR DESCRIPTION
**Pull request by CarlosEduR**

### CODE REVIEW

File: fuzz/validate.cc [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
+  FuzzedDataProvider fdp(data, size);
+  std::string source = fdp.ConsumeRandomLengthString(256);
+  std::string base_source = fdp.ConsumeRandomLengthString(256);
+
+  /**
+   * ada::validate
+   */
+  auto out_aggregator = ada::parse<ada::url_aggregator>(source);
```
Comment (lemire) [inline fuzz/validate.cc:17]: I am somewhat anxious about this code. Are you assuming that 'source' is parsed successfully?

Comment (CarlosEduR) [reply fuzz/validate.cc:17]: > I am somewhat anxious about this code. Are you assuming that 'source' is parsed successfully?

good point @lemire, I'll update it so it only validates the parsed successfully ones.

Comment (CarlosEduR) [reply fuzz/validate.cc:17]: Code updated.

File: fuzz/validate.cc [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  if (out_aggregator) {
+    out_aggregator->validate();
```
Comment (anonrig) [inline fuzz/validate.cc:19]: We can just move this to parse.cc no need to create a new fuzzer

**Review by lemire — APPROVED** [review]

File: fuzz/validate.cc (hunk 2 above)
Comment (CarlosEduR) [reply fuzz/validate.cc]: code updated.

**Review by anonrig — APPROVED** [review]

**Pull request merged**